apps/api-extractor/src/generators/DtsRollupGenerator.ts

```diff
@@ -45,6 +45,35 @@ export enum DtsRollupKind {
   PublicRelease
 }
 
+/**
+ * Compute a relative path between two absolute paths.
+ */
+function computeRelativePath(fromPath: string, toPath: string): string {
```

Comment:
Can you simply use [path.relative()](https://nodejs.org/api/path.html#path_path_relative_from_to)?

Reply:
In this instance, sure. The `Path` module in the node-core-library package in this repo made me think that there was some reason for avoiding it.

Reply:
`FileSystem` is a complete replacement for `fs`, because the native `fs` library was designed as low-level primitives that are not suitable for everyday usage.

`Path` contains supplementary stuff that merely extends the native `path`, since that API is fine to use.

We should probably document this better.